.travis.yml

```diff
@@ -11,3 +11,4 @@ script:
   # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
   - ansible-playbook --syntax-check playbooks/byo/config.yml
   - cd utils && make ci
+  - make test-integration
```

Comment:
make test-integration is running from the utils directory, probably want to do something like the following instead:
```
- ansible-playbook --syntax-check playbooks/byo/config.yml
- pushd utils
- make ci
- popd
- make test-integration
```